Source/Core/Common/CommonFuncs.h

```diff
@@ -5,6 +5,12 @@
 
 #include <optional>
 #include <string>
+#ifdef _WIN32
+#include <SetupAPI.h>
+#include <cfgmgr32.h>
+#include <devpropdef.h>
+#endif
```

Comment:
Please don't do this, Windows headers in .h files cause nothing but headaches. This breaks compilation without pch...